index.d.ts

```diff
@@ -0,0 +1,432 @@
+// Type definitions for ldclient-node
+
+/**
+ * The LaunchDarkly Node.js client interface.
+ *
+ * Documentation: http://docs.launchdarkly.com/docs/node-sdk-reference
+ */
+
+declare module 'ldclient-node' {
+  /**
+   * The LaunchDarkly static global.
+   */
+  export function init(key: string, user: LDUser, options?: LDOptions): LDClient;
```

Comment:
Is the `user: LDUser` param meant to be here?
It doesn't seem to align with `new_client` fn ([exported as `init`](https://github.com/launchdarkly/node-client/blob/142a7dd1d78bf10c31d7df1f7fb454ff19664b3c/index.js#L286)): https://github.com/launchdarkly/node-client/blob/142a7dd1d78bf10c31d7df1f7fb454ff19664b3c/index.js#L19